cmds/core/yes/yes_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+func TestMain(m *testing.M) {
+	testutil.Run(m, main)
+}
+
+func TestYes(t *testing.T) {
+	cmd := testutil.Command(t)
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		t.Fatalf("cmd.StdoutPipe: got %v, want nil", err)
+	}
+
+	if err := cmd.Start(); err != nil {
+		t.Fatalf("cmd.Start: got %v, want nil", err)
+	}
+
+	in := bufio.NewScanner(stdout)
+
+	for i := 0; i < 1000; i++ {
+		if !in.Scan() {
+			t.Fatalf("Could not scan: got %v error, want no error", in.Err())
+		}
+		if text := in.Text(); text != "y" {
+			t.Errorf("Expected a 'y', but received %v", text)
+			break
+		}
+	}
+	if err := cmd.Process.Kill(); err != nil {
+		t.Errorf("Killing the child process: received %v, but wanted nil", err)
```

Comment:
got %v, want nil
The reason to fix on this format is then it's easy to look for all these types of prints:
git grep got.*want

Reply:
Fixed :)! I will get used to this format!